mysql-test/suite/innodb/t/percona_doublewrite.test

```diff
@@ -138,6 +138,64 @@ SET GLOBAL innodb_fast_shutdown=2;
 --let $restart_parameters=restart:--innodb-force-recovery=6
 --source include/start_mysqld.inc
 
+--echo #
+--echo # Bug PS-3411: Parallel doublewrite buffer file created when skip-innodb_doublewrite is set
+--echo #
+
+--let $restart_hide_args= 1
+--let $restart_parameters= restart:--innodb-parallel-doublewrite-path=$DOUBLEWRITE_FILE
+--source include/restart_mysqld.inc
+file_exists $DOUBLEWRITE_FILE;
+
+--disable_warnings
+DROP TABLE IF EXISTS t1;
```

Comment:
Code like this is present in some old tests, but it's not used anymore: if it exists, drop it (without if exists, without disabling warnings), if it doesn't, delete this block.